client/my-sites/plan-features/header.jsx

```diff
@@ -83,7 +95,9 @@ export class PlanFeaturesHeader extends Component {
 				</div>
 				<div className="plan-features__pricing">
 					{ this.getPlanFeaturesPrices() } { this.getBillingTimeframe() }
-					{ this.getIntervalDiscount() }
+					{ isJetpack && abtest( 'onlyJetpackMonthly', countryCode ) === 'original'
```

Comment:
`getIntervalDiscount` already makes sure this is rendered for Jetpack sites only, so I don't think we need an additional `isJetpack` here.

Reply:
:+1: